ietf/nomcom/views.py

```diff
@@ -57,7 +57,7 @@ def index(request):
         else:
             nomcom.url = None
         if year >= 2002:
-            nomcom.ann_url = "/nomcom/ann/#%4d" % year
+            nomcom.ann_url = "/nomcom/ann/#nomcom-%4d" % year
```

Comment:
We'll go with this for now, but consider dropping the hyphen since nomcom2021 is a group.